docs/modules/ROOT/pages/how-tos/install/gcp.adoc

```diff
@@ -148,6 +156,8 @@ Prerequisites:
 * GCP project setup according to above documentation
 * Service Account Key (key.json)
 
+TIP: https://github.com/projectsyn/purser[Purser] can be used to check if the preconditons are met.
```

Comment:
I would wait a bit before including this one. For me it's still unclear if we have to invent this cloud-checker tool ourselves...

Reply:
Conclusion from a discussion between @tobru, @srueg and me:

Even if we find a utility built for the same/a similar use case, it is likely that we need something to make usage easy. Purser can/will be that wrapper. We agreed on keeping it mentioned here.